ecommerce-product-page-main/styled.css

```diff
@@ -116,37 +118,34 @@ body{
     }
 
     .dropdown{
-
         visibility: hidden;
     }
 
 
-        .button2{
-            margin-top: 20px;
-        }
 }
 
+.containflex{
+    display: flex;
+}
 
 
 .container{
-
+    padding-left: 20px;
+    padding-right: 20px;
     display:flex;
-    padding-bottom: 15px;
-    padding-top: 30px;
+    justify-content: space-between;
+    border-bottom: 1px solid #ccc;

```

Comment:
This was to get that  black line at the bottom of the navigation menu right? ,the one i used the "hr" tag for instead.

Reply:
yes